website/components/top/GetStarted.tsx

```diff
@@ -244,9 +244,9 @@ export const GetStarted = () => (
           </Button>
         </Card>
         <Card
-          description="Marpit, independented from Marp, is the skinny framework to transform Markdown + CSS theme to the deck composed of HTML + CSS. It has designed to output only minimum assets."
+          description="Marpit (independented from Marp) is the framework that transforms Markdown and CSS themes to slide decks composed of HTML/CSS. It is optimized to output only the minimum set of assets required."
           name="Marpit framework"
-          summary="The skinny framework for creating slide deck from Markdown"
+          summary="The lightweight framework for creating slide decks from Markdown"
```

Comment:
"The skinny framework" represents the concept of Marpit framework and we want not to change this. I think "lightweight" cannot describe the advantage of the framework (minimum outputs).